### PR TITLE
Add: martinchrzan.ColorPicker version 1.1.1.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/22461)

### CODE REVIEW

**Comment by vedantmgoyal2009** [comment]
@OfficialEsco please see if this PR is correct.

File: manifests/m/martinchrzan/ColorPicker/1.1.1.0/martinchrzan.ColorPicker.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+PackageIdentifier: martinchrzan.ColorPicker
+PackageVersion: 1.1.1.0
+PackageLocale: en-US
+Publisher: Martin Chrzan
+PublisherUrl: https://github.com/martinchrzan
+PublisherSupportUrl: https://github.com/martinchrzan/ColorPicker/issues
+#PrivacyUrl: 
+Author: martinchrzan
+PackageName: ColorPicker
```
Comment (OfficialEsco) [inline manifests/m/martinchrzan/ColorPicker/1.1.1.0/martinchrzan.ColorPicker.locale.en-US.yaml:12]: Looks correct other than this
```suggestion
PackageName: Color Picker
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Comment by vedantmgoyal2009** [comment]
I think now I can proceed with creating manifests for other versions.

**Review by OfficialEsco — APPROVED** [review]

**Comment by vedantmgoyal2009** [comment]
@msftbot Hey, wake up now, it has been a long time you have been sleeping.

File: manifests/m/martinchrzan/ColorPicker/1.1.1.0/martinchrzan.ColorPicker.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Created using YamlCreate.ps1 v1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: martinchrzan.ColorPicker
+PackageVersion: 1.1.1.0
```
Comment (ImJoakim) [inline manifests/m/martinchrzan/ColorPicker/1.1.1.0/martinchrzan.ColorPicker.installer.yaml:5]: image

```suggestion
PackageVersion: 1.1.1
```

File: manifests/m/martinchrzan/ColorPicker/1.1.1.0/martinchrzan.ColorPicker.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Created using YamlCreate.ps1 v1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: martinchrzan.ColorPicker
+PackageVersion: 1.1.1.0
```
Comment (ImJoakim) [inline manifests/m/martinchrzan/ColorPicker/1.1.1.0/martinchrzan.ColorPicker.locale.en-US.yaml:5]: image

```suggestion
PackageVersion: 1.1.1
```

File: manifests/m/martinchrzan/ColorPicker/1.1.1.0/martinchrzan.ColorPicker.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using YamlCreate.ps1 v1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: martinchrzan.ColorPicker
+PackageVersion: 1.1.1.0
```
Comment (ImJoakim) [inline manifests/m/martinchrzan/ColorPicker/1.1.1.0/martinchrzan.ColorPicker.yaml:5]: image

```suggestion
PackageVersion: 1.1.1
```

**Comment by ImJoakim** [comment]
Remember to update the version folder as well 👍

**Comment by vedantmgoyal2009** [comment]
Install MSI once and then check 👍

**Comment by OfficialEsco** [comment]
@ImJoakim we are going to rework some stuff here once Manifest Schema 1.1.0 is out, IIRC winget-cli "ignores" the .0 in the Version

**Comment by ImJoakim** [comment]
> Install MSI once and then check. 👍

Ah, I see. That's an odd way of doing it from the publisher.

**Comment by OfficialEsco** [comment]
We have the same issue with Microsoft Teams and GitHub Desktop

**Comment by ImJoakim** [comment]
Alrighty

**Pull request merged**